aperturedb/EntityDataCSV.py

```diff
@@ -70,3 +72,75 @@ def getitem(self, idx):
     def validate(self):
         if self.header[0] != ENTITY_CLASS:
             raise Exception("Error with CSV file field: " + ENTITY_CLASS)
+
+# Used when a csv has a single entity type that needs to be deleted
+
+
+class EntityDeleteDataCSV(CSVParser.CSVParser):
+    """**ApertureDB Entity Delete Data.**
+
+    This class loads the Entity Data which is present in a csv file,
+    and converts it into a series of aperturedb deletes.
+
+    :::note::
+    Expects a csv file with the following columns:
+
+        ``constraint_PROP1``
+    :::
+
+    Example csv file::
+
+        constraint_id
+        321423532
+        42342522
+        ...
+
+    Example usage:
+
+   ```python
+
+        data = ImageDeleteDataCSV("/path/to/UnusedImages.csv")
```

Comment:
This example should be for  EntityDeleteDataCSV

Reply:
https://github.com/aperture-data/aperturedb-python/issues/266